ui/src/api/hardware-object.js

```diff
@@ -8,6 +8,10 @@ export function sendExecuteCommand(objectType, objectId, command, value) {
     .safeJson();
 }
 
+export function sendExecuteCommandRaw(objectType, objectId, command, value) {
```

Comment:
I'm not super happy with the name `sendExecuteCommandRaw` but I did not come up with something better at the moment, please feel free to suggest :) ?